.travis/before_install.sh

```diff
@@ -39,21 +39,15 @@ cd /tmp || die
 [ $TRAVIS_OS_NAME != linux ] || {    
     sudo apt-get update || die
 
-    sudo -H pip install --upgrade pip || die
     pip install --upgrade pip || die
```

Comment:
In general, use of `sudo` with pip is discouraged.  Also, Travis seems to indicate that `sudo` is not required for the test scripts.